packages/tpetra/core/example/BlockCrs/README.md

```diff
@@ -30,7 +30,7 @@ cmake \
   -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES:BOOL=OFF \
   -D Trilinos_ENABLE_Tpetra:BOOL=ON \
   -D Kokkos_ENABLE_CXX11_DISPATCH_LAMBDA:BOOL=ON \  
-  -D Kokkos_ENABLE_OpenMP:BOOL=${USE_OPENMP} \
+  -D Kokkos_ENABLE_OPENMP:BOOL=${USE_OPENMP} \
   -D Kokkos_ENABLE_Cuda_UVM:BOOL=${USE_CUDA} \
```

Comment:
Any particular reason why `Cuda` and not `CUDA` here?

Reply:
No, that should be CUDA. In order to bootstrap the promotion process, I still allow the deprecated option names so most scripts will continue to work in the short term. This should be changed since we will soon move from deprecation warnings to errors.